sparql/sparql12/syntax-triple-terms-negative/list-anonreifier-01.rq

```diff
@@ -0,0 +1,9 @@
+PREFIX ex: <http://example.com/>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+
+SELECT * WHERE {
+    <<?s ?p ?o>> ex:author ex:Bob .
+    << ex:Moscow a ex:City >> ex:author ?person .
+    # TODO: See if this should be throwing an error
+    << _:b ex:x () >> ex:broken true .
```

Comment:
Is the suggestion here that `()` shouldn't parse as `rdf:nil` and be disallowed because you shouldn't be able to reify a triple with a list as an object?

Reply:
This test was derived from the RDF-star tests.
I suspect you are right.
But I wonder if we really want/need to disallow this.
A [message on this exact topic](https://lists.w3.org/Archives/Public/public-rdf-star-wg/2024Oct/0042.html) was also just raised on the mailinglist.

@afs perhaps you have some more insights on this?

Reply:
See also https://github.com/w3c/rdf-ucr/issues/26 ("Example 3: RDF Lists").

Reply:
See also https://github.com/w3c/rdf-ucr/issues/26 ("Example 3: RDF Lists").